### PR TITLE
[Minor] docs: Add more details about dropping schema and tables

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

- Add a description about parameter `cascade` when dropping the schema 
- Please provide more details on how to drop the table and clarify `dropTable` and `purgeTable` 

### Why are the changes needed?

Make the document more user-friendly.

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

File: docs/manage-metadata-using-gravitino.md [hunk 1]
```diff
@@ -561,6 +561,9 @@ supportsSchemas.dropSchema(NameIdentifier.of("metalake", "catalog", "schema"), t
 </TabItem>
 </Tabs>
 
+If `cascade` is true, Gravitino will drop all tables under the schema. Otherwise, Gravitino will throw an exception if there are tables under the schema. 
+Some catalogs may not support dropping a schema cascading, please refer to the related doc for more details.
```
Comment (jerryshao) [inline docs/manage-metadata-using-gravitino.md:565]: -> dropping a schema cascadingly,

File: docs/manage-metadata-using-gravitino.md [hunk 2]
```diff
@@ -961,9 +964,11 @@ You can remove a table by sending a `DELETE` request to the `/api/metalakes/{met
 <TabItem value="bash" label="Bash">
 
 ```bash
+## purge can be true or false, if purge is true, Gravitino will remove the real data of the table.
```
Comment (jerryshao) [inline docs/manage-metadata-using-gravitino.md:967]: I think you can remove "real", it not necessary to emphasize it.

File: docs/manage-metadata-using-gravitino.md (hunk 1 above)
Comment (yuqi1129) [reply docs/manage-metadata-using-gravitino.md]: Fixed

File: docs/manage-metadata-using-gravitino.md (hunk 2 above)
Comment (yuqi1129) [reply docs/manage-metadata-using-gravitino.md:967]: Fixed

Comment (jerryshao) [reply docs/manage-metadata-using-gravitino.md:967]: Did you fix it?

Comment (yuqi1129) [reply docs/manage-metadata-using-gravitino.md:967]: Oh, I removed the data in line 992. Line 967 should also do it.

Comment (yuqi1129) [reply docs/manage-metadata-using-gravitino.md]: done

**Review by jerryshao — APPROVED** [review]

**Pull request merged**